_format/1.1/index.md

```diff
@@ -1800,12 +2134,14 @@ keyed by `errors` in the top level of a JSON API document.
 An error object **MAY** have the following members:
 
 * `id`: a unique identifier for this particular occurrence of the problem.
+* `type`: a URI that identifies the type of error that this particular error 
+  is an instance of. When dereferenced, this URI **SHOULD** provide a 
+  human-readable explanation of the general error.
 * `links`: a [links object][links] containing the following members:
   * `about`: a [link][links] that leads to further details about this
     particular occurrence of the problem.
 * `status`: the HTTP status code applicable to this problem, expressed as a
   string value.
-* `code`: an application-specific error code, expressed as a string value.
```

Comment:
I'm actually 100% with you that `code` is less useful than `type` ([we've had our struggles](https://www.drupal.org/project/jsonapi/issues/2934362)), but can this be removed during the 1.0 lifecycle?

Edit: after reading below, I see the answer to this. However, perhaps this should be kept here with an additional caveat like: "this member is no longer recommended for use, however some implementations may continue to send this key. Implementors **SHOULD** use the `type` key for new APIs.